symforce/__init__.py

```diff
@@ -104,7 +104,7 @@ def _find_symengine() -> ModuleType:
 
     symengine_path_candidates = list(
         symengine_install_dir.glob("lib/python3*/site-packages/symengine/__init__.py")
-    )
+    ) + list(symengine_install_dir.glob("local/lib/python3*/dist-packages/symengine/__init__.py"))
```

Comment:
For some reason when running the workflow with `python3.10` symengine was installed into `local/lib/python3.10/dist-packages` rather than `/lib/python3.10/site-packages`. I don't actually know what is causing the change in the install location, but it was simple enough to check this other location, so I thought there'd be no harm in just doing so rather than getting to the bottom of the change.

Reply:
I'd guess it's a pip or setuptools version thing?  Not sure though.  I think this is fine